src/allmydata/scripts/runner.py

```diff
@@ -223,21 +290,43 @@ def write_coverage_data():
     reactor.addSystemEventTrigger('after', 'shutdown', write_coverage_data)
 
 
-def _run_with_reactor(reactor):
+def _run_with_reactor(reactor, config, argv, stdout, stderr):
+    """
+    Run a Tahoe-LAFS node using the given reactor.
+
+    :param reactor: The reactor to use.  This implementation largely ignores
+        this and lets the rest of the implementation pick its own reactor.
+        Oops.
```

Comment:
So what *does* this function do differently then? Can it just be merged with `run()`?

Reply:
It serves as a starting point to start respecting the reactor parameter, at least.  From this point inward we *could* be using that reactor, if we wanted.  `run` is the thing that picks a value for that reactor.